dev-resources/tla-demo/docker-compose.yml

```diff
@@ -0,0 +1,211 @@
+# Example Architecture with Two Noisy LRS' and one Transactional LRS
+
+version: "3.9"
+
+volumes:
+  nsy_lrs_1_db_data:
+  nsy_lrs_2_db_data:
+  nsy_lrs_3_db_data:
+  txn_lrs_db_data:
+  lrspipe_storage:
+
+services:
+  # Noisy LRS 1
+  # - Noisy LRS 1: DB
+  nsy_lrs_1_db:
+    image: postgres:14
+    volumes:
+      - nsy_lrs_1_db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: lrsql_user
+      POSTGRES_PASSWORD: lrsql_password
+      POSTGRES_DB: lrsql_db
```

Comment:
Does this PG DB not have its port exposed? I'm assuming it's because it doesn't need to be access outside the Docker stack.

Reply:
nope. the noisy LRSs don't really need their DBs examined. Within the docker ecosystem the app can see them just fine, exposing them would just be to the host computer.